openmrs/apps/reports/reports.json

```diff
@@ -1956,6 +1956,12 @@
                     "config": {
                         "sqlPath": "/var/www/bahmni_config/openmrs/apps/reports/HIV_TESTING_MONTHLY_INDICATOR_REPORT/sql/indicator1_CDV_TS_number_of_people_tested_in_the_month_disaggregation_entry_point.sql"
                     }
+                },{
+                    "name": "Report3a",
```

Comment:
@tresorluz Can we use "Indicator3B" here ? The naming that I am suggesting is "Indicator", followed by its number, and finally A - for age disaggregation, and B - for entry point disaggregation